src/index.html

```diff
@@ -6,7 +6,41 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <main class="page__wrapper">
+      <div class="calendar calendar--start-day_sun calendar--month-length_28">
```

Comment:
Let's follow BEM naming convention
`block--modifier-name--modifier-value`
```suggestion
      <div class="calendar calendar--start-day--sun calendar--month-length--28">
```

Reply:
Дякую, виправив :)